app/command_loadtest.go

```diff
@@ -66,16 +66,17 @@ type LoadTestProvider struct {
 }
 
 func init() {
-	if !utils.Cfg.ServiceSettings.EnableTesting {
-		RegisterCommandProvider(&LoadTestProvider{})
-	}
```

Comment:
1. The configuration is not loaded when `init` is called.
2. The condition is wrong. We want the command to be present when `EnableTesting` is `true`.

Reply:
That doesn't sound good. Did that allow the testing command to be ran when it shouldn't have been enabled?

Reply:
You were able to run it on any server, but on servers without `EnableTesting`, it just wouldn't do anything. Try it on pre-release. No errors like you would get with a command that doesn't exist. But no response either.